internal/server/v1/observations/golden/bulk_series_linked/all_FRA_AA2_2016.json

```diff
@@ -211,233 +211,6 @@
             }
           ]
         },
-        {
-          "entity": "nuts/FR10",
```

Comment:
why is this deletion/diff?

Reply:
This is due to CL/638945277 in which the NUTS<N> to AA<N> bulk change was reverted.
Team is mapping the AdministrativeArea as per the AA level specified in Wikipedia for individual countries as part of boundary addition now.